packages/flutter_test/lib/src/window.dart

```diff
@@ -51,155 +55,170 @@ import 'package:flutter/foundation.dart';
 ///   * [TestPlatformDispatcher], which wraps a [PlatformDispatcher] for
 ///     testing purposes and is used by the [platformDispatcher] property of
 ///     this class.
-class TestWindow implements ui.SingletonFlutterWindow {
+class TestWindow implements SingletonFlutterWindow {
```

Comment:
code organization nit: Maybe move the old (soon-to-be-deprecated) class to the end of the file. People opening this file should see the new and shiny stuff first so they (hopefully) use that and not get distracted by the old stuff we don't want them to use anymore.

Reply:
Make sense to me, but I'm going to hold off until reviewing is done so the diffs aren't _completely_ atrocious. They're already pretty messy as is.

Reply:
Done!